website/docs/guides/migration/versions/04-upgrading-to-v1.4.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Upgrading to v1.4 (prerelease)"
+description: New features and changes in dbt Core v1.4
+---
+### Resources
+
+- [Changelog](https://github.com/dbt-labs/dbt-core/blob/1.3.latest/CHANGELOG.md)
+- [CLI Installation guide](/docs/get-started/installation)
+- [Cloud upgrade guide](/docs/dbt-versions/upgrade-core-in-cloud)
+
+## What to know before upgrading
+
+We are committed to providing backward compatibility for all versions 1.x. If you encounter an error upon upgrading, please let us know by [opening an issue](https://github.com/dbt-labs/dbt-core/issues/new).
+
+### For those on versions .20 or .21
+
+As of June 30, 2022, dbt Cloud has ceased all support for dbt Core versions older than v1.0. With the release of v1.4, these older versions have been removed from the docs site version menu as well. It is strongly recommded that anyone on a version lower than 1.1 upgrade as soon as possible.
+Please visit [About dbt Core Versions](/dbt-versions/core) for more information
```

Comment:
I don't think this section makes sense here, in the v1.4 upgrade guide.

Official support for these versions from the Core team ended on Dec 3, 2021 (the day v1.0 was released). Official support for these versions in dbt Cloud ended on June 30, 2022. As maintainers of the docs, we're taking the opportunity of the v1.4 release to also officially remove v0.20 + v0.21 from the docs site, but we could have chosen to do it sooner—there's no specific tie back to v1.4. Let me know if you disagree!

Reply:
If it doesn't make sense to you, it's out! I added it because I'm still learning how our clients read our docs and thought it might help raise awareness and encourage folks to upgrade, but it's 2 pretty old versions so that was a gamble.